Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN apk add --no-cache \
     libffi \
     libpq \
     nginx \
-    python2 \
+    python3 \
```

Comment:
If supervisor requires python2-do we still need to keep python2 here?

Reply:
The `py2-pip` dependency takes care of that.